.env.dev

```
@@ -1,4 +1,8 @@
-API_BASE_URL=https://api.dev.360ghar.com
-SUPABASE_URL=https://YOUR_DEV_SUPABASE_URL
-SUPABASE_ANON_KEY=YOUR_DEV_SUPABASE_ANON_KEY
-ENABLE_ANALYTICS=false
+API_BASE_URL=https://360ghar.up.railway.app/api/v1
+SUPABASE_URL=https://pezjtoivkrmepydqminz.supabase.co
+SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InBlemp0b2l2a3JtZXB5ZHFtaW56Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTExNTQxMjYsImV4cCI6MjA2NjczMDEyNn0.HeTvrm0mGXhQybNAx5DOAnKSOdecWy_pn3NWaJaVr4g
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SUPABASE_ANON_KEY key should go before the SUPABASE_URL key

(UnorderedKey)